Frontend/src/cse-ui-kit/bold-buttons/BoldButton.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { StyledButton, buttonProps } from './BoldButton-Styled';
+
+
+type Props = {
+  children?: React.ReactElement | any;
+  onClick?: (...args: any) => void;
+} & buttonProps;
+
+export default function BoldButton({ children, onClick, ...styleProps }: Props) {
+  return (
+    <StyledButton
+      onClick={onClick}
+      {...styleProps}
+    >
+      {children}
```

Comment:
Instead of taking in children, place the svg directly into here, the reason being when you want to use the bold button you want to use it like:
```
<BoldButton />
```

not

```
<BoldButton>
   <svg/>
</BoldButton>
```